com.coherentsolutions.training.auto.web.pashkovskaya/src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,25 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage{
```

Comment:
let's keep the following structure
class
-- constants
-- locators
-- constructor
-- methods
-- predicates